### ISSUE TITLE
Demo Support Issue for security Team

### ISSUE THREAD

**Issue by FaGru3n** [eclipse-tractusx/sig-infra#5]
**Please specify the type of request. Exemplary requests are:**

**Tooling:**   

* Veracode (initial setup, GitHub integration)
* Trivvy
* KICS
* Gitguardian
* Pen-testing
* Security risk assessment (RRA, Threat modeling, Code reviews)

Info: @the-tatanka  just testing out the options for GitHub projects boards, whats your opinon about it?

**Comment by FaGru3n** [comment]
may you want to add more explainations to the default template 😉 

current markdown looks like this:

``` bash
**Please specify the type of request. Exemplary requests are:**

**Tooling:**   

* Veracode (initial setup, GitHub integration)
* Trivvy
* KICS
* Gitguardian
* Pen-testing
* Security risk assessment (RRA, Threat modeling, Code reviews)
```

**Comment by FaGru3n** [comment]
will be implemented with #8
